test/suites/localzone/suite_test.go

```diff
@@ -0,0 +1,81 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package localzone_test
+
+import (
+	"testing"
+
+	"github.com/samber/lo"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+
+	corev1beta1 "sigs.k8s.io/karpenter/pkg/apis/v1beta1"
+	"sigs.k8s.io/karpenter/pkg/test"
+
+	"github.com/aws/karpenter-provider-aws/pkg/apis/v1beta1"
+	"github.com/aws/karpenter-provider-aws/test/pkg/environment/aws"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var env *aws.Environment
+var nodeClass *v1beta1.EC2NodeClass
+var nodePool *corev1beta1.NodePool
+
+func TestLocalZone(t *testing.T) {
+	RegisterFailHandler(Fail)
+	BeforeSuite(func() {
+		env = aws.NewEnvironment(t)
+	})
+	AfterSuite(func() {
+		env.Stop()
+	})
+	RunSpecs(t, "Chaos")
+}
+
+var _ = BeforeEach(func() {
+	env.BeforeEach()
+	nodeClass = env.DefaultEC2NodeClass()
+	// local zones don't support gp3 volumes, use gp2 instead
```

Comment:
I assume this should change eventually. Worth a note with a reference to know when we can remove this?

Reply:
Probably, I'd have to check if it's on a roadmap somewhere. If it is I'll link that here.

Reply:
Looks like this isn't actually the case in LAX where we'll be testing.
> Q: What EBS volume types are available in AWS Local Zones?
 Local Zones in Los Angeles (us-west-2-lax-1a, and us-west-2-lax-1b), Dallas (us-east-1-dfw-2a), and  Phoenix (us-west-2-phx-2a) provide gp2, gp3, io1, st1, and sc1 EBS volumes. All other Local Zones provide gp2 volume.

I've not been able to find any roadmap for wider gp3 support but I will add a not and link to the existing documentation about what local zones support what. I feel like it does make sense to leave this in though even though we technically don't need to since we're testing in LAX. This way ensures we have some flexibility about what local zones we're running in.